### PR TITLE
Alargean/compose sample updates

### PR DESCRIPTION
**Pull request by AlinArgeanu**
## 📦 Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Enhancement to a current functionality
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Bug fix (non-breaking change which fixes an issue. Please add issue link here too).

## ✅ Checklist

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] This PR contains new tests that cover the new code.
- [ ] This PR refactor previous tests to cover the new code.
- [ ] This PR is part of a set of PRs that build a bigger feature. If so, please, add here links to previously merged or open PRs.

### CODE REVIEW

**Comment by joyl1216** [comment]
Hi @AlinArgeanu Good job on the sample updates, especially the new `SliderState`. 👍 There is an existing issue with the `Companion Pane sample`, which should not show the top bar on the bottom screen in the dual-landscape mode. And the dark theme probably needs to be disabled since it doesn't fit the new light-blue theme color.

The updated Compose sample(better be updated to match the following one)
Screenshot_1657748460
Screenshot_1657748460

The current Kotlin sample
Screenshot_1657748434
Screenshot_1657748434

File: DualView/src/main/java/com/microsoft/device/display/samples/dualview/ui/theme/Theme.kt [hunk 1]
```diff
@@ -13,15 +13,15 @@ import androidx.compose.runtime.Composable
...
 )
 
 private val LightColorPalette = lightColors(
-    primary = red200,
-    primaryVariant = red500,
+    primary = lightBlue,
+    primaryVariant = lightBlue,
     secondary = gray500,
     onPrimary = Color.White,
 )
```
Comment (khalp) [inline DualView/src/main/java/com/microsoft/device/display/samples/dualview/ui/theme/Theme.kt:27]: I think the Dual View sample in the bundle app only has light mode, so maybe we can delete the dark theme and only use the light color palette

File: TwoPage/src/main/java/com/microsoft/device/display/samples/twopage/ui/theme/Theme.kt [hunk 2]
```diff
@@ -5,17 +5,20 @@ import androidx.compose.material.MaterialTheme
...
 )
 
 private val LightColorPalette = lightColors(
-    primary = Purple500,
-    primaryVariant = Purple700,
-    secondary = Teal200
+    primary = lightBlue,
+    primaryVariant = lightBlue,
+    secondary = Teal200,
+    onPrimary = Color.White,
```
Comment (khalp) [inline TwoPage/src/main/java/com/microsoft/device/display/samples/twopage/ui/theme/Theme.kt:21]: same for TwoPage, I think the bundle app version only has the light color palette

File: TwoPage/src/main/java/com/microsoft/device/display/samples/twopage/ui/view/HomePage.kt [hunk 3]
```diff
@@ -9,25 +9,61 @@ import androidx.compose.foundation.layout.fillMaxHeight
...
+                foldSizeDp = windowState.foldSizeDp
+            )
+        }
+    )
+/*    TwoPageAppContent(
         pane1WidthDp = windowState.pane1SizeDp.width,
         pane2WidthDp = windowState.pane2SizeDp.width,
         isDualScreen = windowState.isDualPortrait(),
         foldSizeDp = windowState.foldSizeDp
+    )*/
```
Comment (khalp) [inline TwoPage/src/main/java/com/microsoft/device/display/samples/twopage/ui/view/HomePage.kt:51]: I think you can remove the commented out code

**Review by khalp — COMMENTED** [review]
All the theme updates, icons, and slider state changes look great, thanks!

Would you also be able to take new app screenshots and update the READMEs? we may eventually want to update the [samples documentation](https://docs.microsoft.com/en-us/dual-screen/android/jetpack/compose/samples) page as well using these new screenshots

File: CompanionPane/src/main/java/com/microsoft/device/display/samples/companionpane/ui/components/SliderState.kt [hunk 4]
```diff
@@ -0,0 +1,87 @@
+package com.microsoft.device.display.samples.companionpane.ui.components
```
Comment (bimiron) [inline CompanionPane/src/main/java/com/microsoft/device/display/samples/companionpane/ui/components/SliderState.kt:1]: I think we should add the copyright for this file as well

**Comment by joyl1216** [comment]
@AlinArgeanu I think the PR is almost ready to merge. Just one more thing, apart from the document, the screenshots in the `README`s here also need to be updated. Would you please include them in this PR?

**Review by khalp — APPROVED** [review]
Changes look awesome, thank you for the great work!!

If you have time to update one more thing, it would be great to update the app icons in the [main repo README](https://github.com/microsoft/surface-duo-compose-samples/tree/alargean/compose-sample-updates#readme) to reflect the new icons you just added 😄 
image

**Comment by joyl1216** [comment]
@AlinArgeanu thanks for your awesome work!

**Review by bimiron — APPROVED** [review]
Nice work!

**Pull request merged**